README.md

```diff
@@ -29,21 +29,6 @@ cp example-config.js my/folder/config.js
 Then edit that config file and enter your settings. Read the comments in the
 file for further details about each option.
 
-## Migration to v2
```

Comment:
Moved this section to closer to the bottom of the doc - it felt like old / arcane instructions that is better avoided as the first thing a user sees. (Maybe it's worth deleting altogether at this point?)

Reply:
I am still afraid to delete it, but we are getting there soon. Thanks!